src/index.js

```diff
@@ -0,0 +1,25 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+import JSON5 from 'json5';
+
+function Json5Loader(source) {
+  if (this.cacheable) {
+    this.cacheable();
+  }
+
+  let value;
+
+  try {
+    value = JSON5.parse(source);
+  } catch (e) {
+    throw new Error('Error using JSON5 parsing');
+  }
+
+  this.values = [value];
```

Comment:
This can be dropped aswell, if I remember right (webpack =< v0.6 or something like that :))